Regression/WarpX-tests.ini

```
@@ -1937,6 +1937,21 @@ analysisRoutine = Examples/Tests/collision/analysis_collision_rz.py
 aux1File = Regression/PostProcessingUtils/post_processing_utils.py
 tolerance = 1.e-15
 
+[Proton_Boron_Fusion_3D]
+buildDir = .
+inputFile = Examples/Modules/nuclear_fusion/inputs_3d
+runtime_params =
+dim = 3
+addToCompileString =
```

After #2556 was merged:
```suggestion
addToCompileString =
cmakeSetupOpts = -DWarpX_DIMS=3
```